### PR TITLE
add function to copy default configuration from package

### LINKED ISSUE #41: streamline getting example configuration
Currently you must clone the repository to get example config from the `configs` directory to be able to actually run the server. This process needs to be streamlined to remove this step.

What I'm thinking for implementing this:

- store example configs somewhere like in `feature_server` to be copied into the module/package
- add a cli argument like `--copy-config` to copy the default configuration to `~/.piqueserver/` or a custom location
- make sure the server knows the default location to read configuration from - it should be the dot directory in the user's home directory - don't simply use the example config silently as installed from the package (security risk)

### PR DESCRIPTION
**Pull request by swalladge**
Changes in this pull request:
 * move example configuration into `feature_server/` to be installed along with the package
 * add cli argument `--copy-config` which copies the default configuration to one of the following locations (in order of precedence):
   * the directory specified by the `-d` option
   * `$XDG_CONFIG_DIR/piqueserver/`
   * `~/.config/piqueserver/`

Fixes #41

### CODE REVIEW

**Comment by swalladge** [comment]
might want to squash merge this one to avoid the accidently added and removed log files...

**Comment by noway421** [comment]
oohh, good point

**Comment by noway421** [comment]
haven't looked at it yet though

**Comment by swalladge** [comment]
I also should add that this simplifies the initial installation process to:

```
# install
pip install piqueserver

# copy the default config
piqueserver --copy-config

# and run!
piqueserver
```

**Comment by feikname** [comment]
Thank god this will be squash merged lol. Fucked up a bit.

File: .gitignore [hunk 1]
```diff
@@ -29,8 +29,8 @@ dist/
 
 # config/userdata
 configs/maps/*.txtc
-configs/logs/
-configs/data/
+configs/logs/*
+configs/data/*
```
Comment (noway421) [inline .gitignore]: config instead of configs?

Comment (noway421) [inline .gitignore]: fixed in master

**Pull request merged**